core/ir/aarch64/codec.c

```diff
@@ -226,8 +226,11 @@ encode_pc_off(OUT uint *poff, int bits, byte *pc, instr_t *instr, opnd_t opnd,
         *poff = off >> 2 & (range - 1);
         return true;
     }
-    /* If !di->check_reachable we still require correct alignment. */
```

Comment:
Besides the label instr case, does this comment still hold true?

Reply:
The code no longer checks for alignment for `!check_reachable` so no the comment should not be there as it would not match the code.

Or do you mean, should the code change?  I would argue no: `!check_reachable` means we should not care about the target at all, in my book.  It is set prior to the final encoding when various pieces are not in place to know whether the target is valid.

Reply:
> Or do you mean, should the code change?

Yeah, I was just trying to understand why the comment says we still require correct alignment even if `!di->check_reachable`.

> I would argue no: !check_reachable means we should not care about the target at all, in my book.

Okay.